lib/actions/deploy.js

```diff
@@ -0,0 +1,331 @@
+/**
+ * @license
+ * Copyright 2016-2020 Balena Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Imported here because it's needed for the setup
+// of this action
+import * as Promise from 'bluebird';
+
+import * as dockerUtils from '../utils/docker';
+import * as compose from '../utils/compose';
+import { registrySecretsHelp } from '../utils/messages';
+import { ExpectedError } from '../errors';
```

Comment:
I removed a `({ ExpectedError } = require('../errors'))` require as it's imported at the top level here